uber/templates/signups/shifts.html

```diff
@@ -116,21 +111,24 @@
             header: {
                 left: 'prev,next today',
                 center: 'title',
-                right: 'agendaDay,listWeek'
-            },
-			buttonText : {
-                today : 'Today',
-                agendaWeek : 'Week',
-                agendaDay : 'Day',
-                listWeek : 'List'
+                right: 'agendaDay,agendaConWeek,listConDuration'
```

Comment:
We replace listWeek with a custom view called agendaConWeek, which is exactly like listWeek but it shows all the days the attendee can sign up for instead of a plain Sun-Sat week. We also add listConDuration, which is a 'list' view of the same.